contracts/src/Teleporter/tests/ReceiptsQueueTests.t.sol

```diff
@@ -9,87 +9,99 @@ import "forge-std/Test.sol";
 import "../ReceiptQueue.sol";
 
 contract ReceiptQueueTest is Test {
+    using ReceiptQueue for ReceiptQueue.TeleporterMessageReceiptQueue;
     // The state of the contract gets reset before each
     // test is run, with the `setUp()` function being called
     // each time after deployment.
-    ReceiptQueue public queue;
+    ReceiptQueue.TeleporterMessageReceiptQueue private _queue;
 
-    function setUp() public virtual {
-        queue = new ReceiptQueue();
-    }
+     // Add 3 elements to the queue.
+    TeleporterMessageReceipt private _receipt1 = TeleporterMessageReceipt({
+        receivedMessageID: 543,
+        relayerRewardAddress: 0x10eB43ef5982628728E3E4bb9F78834f67Fbb40b
+    });
+    TeleporterMessageReceipt private _receipt2 = TeleporterMessageReceipt({
+        receivedMessageID: 684384,
+        relayerRewardAddress: 0x10eB43ef5982628728E3E4bb9F78834f67Fbb40b
+    });
+    TeleporterMessageReceipt private _receipt3 = TeleporterMessageReceipt({
+        receivedMessageID: 654351,
+        relayerRewardAddress: 0xcC8E718045817AebA89592C72Ae1C9917f5D0894
+    });
 
     function testReceiptQueue() public {
```

Comment:
Could we renaming this test case `testEnqueueDequeueSuccess` to differentiate it now that there are other test cases for the `ReceiptQueue`?